src/main/java/com/moddy/server/service/model/ModelRetrieveService.java

```diff
@@ -0,0 +1,28 @@
+package com.moddy.server.service.model;
+
+import com.moddy.server.controller.model.dto.response.OpenChatResponse;
+import com.moddy.server.service.application.HairModelApplicationRetrieveService;
+import com.moddy.server.service.designer.DesignerRetrieveService;
+import com.moddy.server.service.offer.HairServiceOfferRetrieveService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class ModelRetrieveService {
+
+    private final DesignerRetrieveService designerRetrieveService;
+    private final HairServiceOfferRetrieveService hairServiceOfferRetrieveService;
+    private final HairModelApplicationRetrieveService hairModelApplicationRetrieveService;
+
+    public OpenChatResponse getOpenChatInfo(Long userId, Long offerId) {
+        Long designerId = hairServiceOfferRetrieveService.getDesignerApplicationId(offerId).designerId();
+        Long applicationId = hairServiceOfferRetrieveService.getDesignerApplicationId(offerId).applicationId();
+
+        OpenChatResponse openChatResponse = new OpenChatResponse(hairModelApplicationRetrieveService.getApplicationCaptureUrl(applicationId), designerRetrieveService.getDesignerOpenDetail(designerId).kakaoUrl(), designerRetrieveService.getDesignerOpenChatDetail(designerId));
```

Comment:
p2
applicationRetreieveService를 통해서 applicationCaptureUrl을 가져온 것은 좋습니다.
designerResponse를 반환할 때 designerRetrieveService에서 하는 것이 아닌 해당 로직에서 하는 것은 어떠나요?

_아래 suggestion 코드의 변수 네이밍은 임의로 정한 것입니다! 코드를 반영하실 땐 적절한 네이밍으로 작성해주세요!_
```suggestion
        String applicationCaptureUrl = hairModelApplicationRetrieveService.getApplicationCaptureUrl(applicationId);
        DesignerInfoOpenChatDTO dto = designerRetrieveService.getDesignerOpenDetail(designerId);
        DesignerInfoOpenChatResponse response = new DesignerInfoOpenChatResponse(dto.imgUrl(), dto.shopName(), dto.name(), dto.introduction())
        OpenChatResponse openChatResponse = new OpenChatResponse(applicationCaptureUrl, dto.kakaoUrl(), response);
        return openChatResponse;
```